addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/message/L_Message.java

```diff
@@ -18,6 +18,8 @@
 import org.openhab.binding.max.internal.device.DeviceConfiguration;
 import org.slf4j.Logger;
 
+import com.google.common.base.Charsets;
```

Comment:
Since Java 7 there is StandardCharsets please take a look at that, I prefer JDK defaults over non-standard stuff.